### ISSUE TITLE
Ligature support

### ISSUE THREAD

**Issue by antistic** [polybar/polybar#2360]
## Why does polybar need this feature?
It would look nice and would match other titlebars like i3 (see the "fi" ligature mentioned in https://github.com/polybar/polybar/issues/1329#issuecomment-406716144).

The more important issue in my opinion is emoji because many work because of ligatures, such as flags (mentioned previously in https://github.com/polybar/polybar/issues/905#issuecomment-763858560), [gender variants](https://emojipedia.org/woman-technologist/), [skin tone variants](https://emojipedia.org/clapping-hands-dark-skin-tone/), and other joined emoji like [families](https://emojipedia.org/family-man-man-boy-boy/). Not rendering these properly can change the meaning of the symbols entirely.

**Comment by patrick96** [comment]
Yeah, polybar needs better support for these kinds of things. There was also #2062 that requested something along the same lines.

If someone has experience with cairo+pango font rendering, I would greatly appreciate your input here.
